common/changes/@itwin/core-backend/rob-ecsql-tests_2024-12-13-11-28.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "Add ECSql test suite",
```

```suggestion
      "comment": "",
```

End users don't care about tests.